packages/conf-libelf/conf-libelf.1.0.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+authors: ["Bayge <alex@fluidity.money>"]
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+build: ["pkg-config" "--exists" "libelf"]
+depexts: [
+  ["libelf-dev"] {os-family = "debian"}
```

Comment:
```suggestion
  ["libelf-dev"] {os-family = "debian" | os-family = "ubuntu"}
```
Otherwise this is going to fail on
- Ubuntu (see this Matrix: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values) and
- Linux distributions derived from Ubuntu. One example is Linux Mint